fiftyone/server/routes/sort.py

```diff
@@ -25,31 +24,22 @@ async def post(self, request: Request, data: dict):
         filters = data.get("filters", {})
         stages = data.get("view", None)
         dist_field = data.get("dist_field", None)
-        dataset = fod.load_dataset(dataset_name)
 
-        changed = False
-        if dist_field and not dataset.get_field(dist_field):
-            dataset.add_sample_field(dist_field, fof.FloatField)
-            changed = True
-
-        fosv.get_view(dataset_name, stages=stages, filters=filters)
-
-        state = fose.get_state().copy()
         view = fosv.get_view(dataset_name, stages=stages, filters=filters)
-        state.dataset = view._dataset
+        dataset = view._dataset
```

Comment:
Unrelated to the rest of this PR, while inspecting server code, it seemed to me that this route's current implementation was broken...

Reply:
Copying the state and modifying has no effect